apps/dashboard/src/components/create-workflow-button.tsx

```diff
@@ -152,7 +152,11 @@ export const CreateWorkflowButton = (props: CreateWorkflowButtonProps) => {
                       <FormLabel hint={`(max. ${MAX_TAG_ELEMENTS})`}>Add tags</FormLabel>
                     </div>
                     <FormControl>
-                      <TagInput suggestions={tagsQuery.data?.data.map((tag) => tag.name) || []} {...field} />
```

Comment:
❓ How did this work before without the value?

Reply:
not sure, but might not be needed, I'll recheck